htdocs/compta/paiement.php

```diff
@@ -63,6 +63,12 @@
 {
     $socid = $user->socid;
 }
+// list of limit id for invoice
+$facids		= GETPOST('facids','alpha');
```

Comment:
Should use 
GETPOST('facids', 'intcomma');
or a SQL injection will be possible later in the $sql .= ' AND f.rowid IN('.$facids.')';